tests/acceptance/arrays-test.js

```diff
@@ -1,31 +1,27 @@
-import Ember from 'ember';
-import startApp from '../helpers/start-app';
-import { module, test } from 'qunit';
+import { test } from 'qunit';
+import moduleForAcceptance from '../helpers/module-for-acceptance';
 
-var application, store;
-
-module('Acceptance: Arrays Test', {
-  beforeEach: function() {
-    application = startApp();
-    store = application.__container__.lookup('service:simple-store');
+moduleForAcceptance('Acceptance: Arrays Test', {
+  beforeEach() {
+    this.store = this.application.__container__.lookup('service:simple-store');
   },
-  afterEach: function() {
-    Ember.run(application, 'destroy');
+  afterEach() {
+    delete this.store;
```

Comment:
I think you are right.  I've read before delete may lead to memory leaks whereas null will always break references to the object.  So maybe safer to set to null.